src/views/Ladder/Ladder.tsx

```diff
@@ -538,14 +538,34 @@ export class LadderRow extends React.Component<LadderRowProperties, LadderRowSta
 function canChallengeTooltip(obj: any): string {
     if (obj.reason_code) {
         switch (obj.reason_code) {
-            case 0x001: return pgettext("Can't challenge player in ladder because: ", "Can't challenge yourself");
-            case 0x002: return pgettext("Can't challenge player in ladder because: ", "Player is a lower rank than you");
-            case 0x003: return pgettext("Can't challenge player in ladder because: ", "Player is not in the ladder");
-            case 0x004: return pgettext("Can't challenge player in ladder because: ", "Player's rank is too high");
-            case 0x005: return interpolate(pgettext("Can't challenge player in ladder because: ", "Already playing {{number}} games you've initiated"), {"number": obj.reason_parameter });
-            case 0x006: return pgettext("Can't challenge player in ladder because: ", "Already playing a game against this person");
-            case 0x007: return pgettext("Can't challenge player in ladder because: ", "Last challenge within 7 days");
-            case 0x008: return pgettext("Can't challenge player in ladder because: ", "Player already has the maximum number of challenges");
+            case 0x001:
+                return pgettext("Can't challenge player in ladder because: ", "Can't challenge yourself");
+            case 0x002:
+                return pgettext("Can't challenge player in ladder because: ", "Player is a lower rank than you");
+            case 0x003:
+                return pgettext("Can't challenge player in ladder because: ", "Player is not in the ladder");
+            case 0x004:
+                return pgettext("Can't challenge player in ladder because: ", "Player's rank is too high");
+            case 0x005:
+                return interpolate(
+                    pgettext(
+                        "Can't challenge player in ladder because: ",
+                        "Already playing {{number}} games you've initiated",
+                    ),
+                    { number: obj.reason_parameter },  // eslint-disable-line id-denylist
```

Comment:
Here (and elsewhere) I ran `Format Selection`, then added the `eslint-disable` comment.  `Format Selection` caused the whole switch block to be formatted.  Hopefully that explains why the delta on this hunk is so high, even though the salient part is the `// eslint-disable...`